src/lib/color-picker/color-picker-selector.component.ts

```diff
@@ -16,19 +16,20 @@ import {
 } from '@angular/core';
 import { FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms';
 import { BehaviorSubject, Observable, Subscription } from 'rxjs';
+import { EMPTY_COLOR, ENABLE_ALPHA_SELECTOR, parseColorString, toHex, toRgba } from './color-picker';
+import { tinycolor } from '@thebespokepixel/es-tinycolor';
+import type { Instance } from 'tinycolor2';
 import { map } from 'rxjs/operators';
-import { EMPTY_COLOR, ENABLE_ALPHA_SELECTOR, coerceHexaColor, convertRgbToHex, isValidColor } from './color-picker';
+import { MccColorPickerService } from './color-picker.service';
 
-interface ColorOption {
-  type: string;
-  value: number;
-}
 
-interface Offsets {
+interface Coordinates {
   x: number;
   y: number;
 }
 
+const INITIAL_COLOR: Instance = tinycolor({r: 255, g: 0, b: 0, a: 1});
```

Comment:
The red color as default seems to be too aggressive.
I'm thinking that start with white will be a better fit.